Tests/kwarchartTests/kwarchartTests.swift

```diff
@@ -1,7 +1,15 @@
 import XCTest
-@testable import kwarchart_ios
+@testable import kwarchart
 
 final class kwarchart_iosTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+
+        let app = XCUIApplication()
```

Comment:
please add launch argument to identify that we're running the app for the sake of testing ref: https://github.com/jaimejazarenoiii/CartTracker/blob/main/CartTrackerUITests/ShopListViewUITests.swift#L16
https://github.com/jaimejazarenoiii/CartTracker/blob/main/CartTracker/App%20Defaults/AppDelegate.swift#L14